lib/service/Service.ts

```diff
@@ -209,10 +213,11 @@ class Service extends EventEmitter {
   }
 
   /*
-   * Subscribe to incoming peer orders.
+   * Subscribe to incoming peer orders and cancellations.
```

Comment:
For consistency's sake let's have this say `Subscribe to peer order events.` as well.